src/snyk/snykCode/views/suggestion/codeSuggestionWebviewScript.ts

```diff
@@ -6,6 +6,9 @@
 /* eslint-disable @typescript-eslint/no-unsafe-call */
 /// <reference lib="dom" />
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars,@typescript-eslint/no-explicit-any
+declare const acquireVsCodeApi: any;
+
```

Comment:
Is it correct to say that this package could bring the types we need?

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/53370
https://www.npmjs.com/package/@types/vscode-webview

If so, do you think we could start using them?

Reply:
Yes, let's do that, but put that into a different PR